nginx-controller/controller/controller.go

```diff
@@ -1222,6 +1223,30 @@ func (lbc *LoadBalancerController) getMinionsForMaster(master *nginx.IngressEx)
 			glog.Errorf("Ingress Resource %v/%v with the 'nginx.org/mergible-ingress-type' annotation set to 'minion' must contain a Path", ings.Items[i].Namespace, ings.Items[i].Name)
 			continue
 		}
+
+		for key, val := range ings.Items[i].Annotations {
+			if key == "nginx.org/mergible-ingress-type" || !strings.Contains(key, "nginx.org/") {
+				continue
+			}
+			if !strings.Contains(val, "serviceName=") {
+				glog.Errorf("Ingress Resource %v/%v with the 'nginx.org/mergible-ingress-type' annotation set to 'minion' cannot contain any host level annotations",
+					ings.Items[i].Namespace, ings.Items[i].Name)
+				continue IngressLoop
+			}
+		}
+
+		for _, path := range ings.Items[i].Spec.Rules[0].HTTP.Paths {
```

Comment:
Here we will skip any minion with the same path as another minion. Therefore only 1 of the 2 minions will be applied.

Is there any type of order used when listing ingresses using `lbc.ingLister.List()`? Should I apply some type of sorting after the list is generated, so that we know it will always be the first successful minion that will be active?